tests/phpunit/Provider/AutoDiscoverIdTest.php

```diff
@@ -6,8 +6,11 @@
 
 use Inpsyde\App\Container;
 use Inpsyde\App\Provider\RegisteredOnly;
-use Inpsyde\App\Tests\TestCase;
+use PHPUnit\Framework\TestCase;
```

Comment:
is that right? 🤔

Reply:
`Inpsyde\App\Tests\TestCase` extends `PHPUnit\Framework\TestCase`, but this specific case uses nothing from `Inpsyde\App\Tests\TestCase`, so even extending the parent directly is fine. 
Don't change much, but it is fine.

Reply:
Yes, it is a bit leftover from my investigation because I could not repeat the bug initially. But IMO it's nice to don't use monkey/patchwork if possible.